tensorflow/python/keras/engine/training_test.py

```diff
@@ -2022,6 +2022,29 @@ def test_calling_model_on_same_dataset(self):
       model.fit(dataset, epochs=1, steps_per_epoch=2, verbose=0,
                 validation_data=dataset, validation_steps=2)
 
+  def test_training_single_output_model_on_dataset(self):
```

Comment:
Should this test also run in eager mode? (see test decorators on other tests in this file).

Reply:
Please add a comment explaining what behavior this test is testing exactly.